### PR TITLE
Add rq-dashboard flask logging verbosity option

### PR DESCRIPTION
**Pull request by njmei**
This commit adds the `--verbose_logging` option to
the rq-dashboard cli. This option will be set to
false by default. Using this option will result
in verbose werkzeug messages.

e.g. `"GET /index.html HTTP/1.1" 200`

Relates to: #100 and #149

### CODE REVIEW

**Comment by njmei** [comment]
Tests fail, but looks like the failure was introduced in: https://github.com/eoranged/rq-dashboard/commit/bd5266a9d7d52df3980e26d6e2ad18b60f6a4756

https://travis-ci.org/eoranged/rq-dashboard/builds/569717245?utm_source=github_status&utm_medium=notification

File: rq_dashboard/cli.py [hunk 1]
```diff
@@ -106,6 +116,8 @@ def make_flask_app(config, username, password, url_prefix, compatibility_mode=Tr
     '--delete-jobs', default=False, help='Delete jobs instead of cancel')
 @click.option(
     '--debug/--normal', default=False, help='Enter DEBUG mode')
+@click.option(
+    '--verbose-logging', default=False, help='Make Flask logger verbose')
```
Comment (eoranged) [inline rq_dashboard/cli.py]: Could you please also add it to the README?

File: rq_dashboard/version.py [hunk 2]
```diff
@@ -1 +1 @@
-VERSION = '0.5.2'
+VERSION = '0.5.3'
```
Comment (eoranged) [inline rq_dashboard/version.py]: Please don't bump the version in pull-requests. It's usually done in separate commit when the changes in master are stabilized.

**Comment by eoranged** [comment]
About tests.
These are failing in master, I'm currently in the process of stabilizing them, so don't worry for now.

File: rq_dashboard/version.py (hunk 2 above)
Comment (njmei) [reply rq_dashboard/version.py]: Sorry, didn't know! I've undone that change.

File: rq_dashboard/cli.py (hunk 1 above)
Comment (njmei) [reply rq_dashboard/cli.py]: Okay! I've added the new option to the readme.

**Comment by njmei** [comment]
Okay, let me know when master is green again and I can rebase this PR.

**Comment by eoranged** [comment]
Now master is green. Beware that tests will execute FLUSHDB command on Redis instance on default port.
Also, I missed that you are a first-time contributor, so please add your name to AUTORS file :)
As for the rest It's GTM.

**Comment by njmei** [comment]
@eoranged Okay, everything passes now! I ended up moving the log `setLevel` stuff out of the `make_flask_app()` function as it didn't really make sense there. You can merge whenever is convenient for you.

You should also be able to close the two issues I mentioned at the top.

**Comment by eoranged** [comment]
Thanks, Nicholas!
Merged.
I will release It as soon as remaining changes on jobs sorting are done.
